backend/.env.example

```
@@ -0,0 +1,9 @@
+# Supabase PostgreSQL — use transaction pooler (port 6543) for DATABASE_URL
+DATABASE_URL="postgresql://postgres.[project-ref]:[password]@aws-0-[region].pooler.supabase.com:6543/postgres?pgbouncer=true"
+
+# Direct connection (port 5432) for migrations and Prisma Studio
+DIRECT_URL="postgresql://postgres.[project-ref]:[password]@aws-0-[region].pooler.supabase.com:5432/postgres"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)